tests/wpt/metadata/css/css-animations/animationevent-pseudoelement.html.ini

```diff
@@ -1,6 +1,2 @@
 [animationevent-pseudoelement.html]
   bug: https://github.com/servo/servo/issues/10316
```

Comment:
This file (and similar ones referencing 10316) can be removed now.

Reply:
Thanks. I've removed these.